### PR TITLE
docs: add @agroupp as a contributor

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Arthur Groupp is not on the All Contributors list.

Issue Number: N/A

## What is the new behavior?

Arthur Groupp is on the All Contributors list for his idea regarding the Lumberjack Payload.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -774,11 +774,12 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+    <td align="center"><a href="https://indepth.dev/author/layzee/"><img src="https://avatars1.githubusercontent.com/u/6364586?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Lars Gyrup Brink Nielsen</b></sub></a><br /><a href="https://github.com/ngworker/lumberjack/issues?q=author%3ALayZeeDK" title="Bug reports">🐛</a> <a href="https://github.com/ngworker/lumberjack/commits?author=LayZeeDK" title="Code">💻</a> <a href="https://github.com/ngworker/lumberjack/commits?author=LayZeeDK" title="Documentation">📖</a> <a href="#example-LayZeeDK" title="Examples">💡</a> <a href="#ideas-LayZeeDK" title="Ideas, Planning, & Feedback">🤔</a> <a href="#mentoring-LayZeeDK" title="Mentoring">🧑‍🏫</a> <a href="#maintenance-LayZeeDK" title="Maintenance">🚧</a> <a href="#projectManagement-LayZeeDK" title="Project Management">📆</a> <a href="https://github.com/ngworker/lumberjack/pulls?q=is%3Apr+reviewed-by%3ALayZeeDK" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/ngworker/lumberjack/commits?author=LayZeeDK" title="Tests">⚠️</a> <a href="#tool-LayZeeDK" title="Tools">🔧</a> <a href="#userTesting-LayZeeDK" title="User Testing">📓</a></td>
```
Comment (LayZeeDK) [inline README.md:777]: Please keep my link as `https://dev.to/layzee/`. I guess it should be changed in `.all-contributorsrc`.

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NachoVazquez) [reply README.md:778]: Yes, I will fix it in both places!

**Review by SerkanSipahi — APPROVED** [review]
LGTM!

**Review by SerkanSipahi — APPROVED** [review]
LGTM 💯

**Review by santoshyadavdev — APPROVED** [review]
LGTM

**Review by LayZeeDK — APPROVED** [review]

**Pull request merged**